app/components/Footer/index.css

```diff
@@ -56,72 +60,92 @@
-            background-color: transparent;
+        }
+        & .email {
+          background-color: transparent;
+          border: 0;
+          box-sizing: border-box;
+          color: white;
+          font-family: "Palanquin", sans-serif;
+          font-size: 14px;
+          font-weight: 400;
+          width: 100%;
+          &:focus {
             border: 0;
-            box-sizing: border-box;
-            color: white;
-            font-family: "Palanquin", sans-serif;
-            font-size: 14px;
-            font-weight: 400;
-            width: 100%;
-            &:focus {
-              border: 0;
-              outline: 0;
-            }
+            outline: 0;
           }
-          & .email-btn {
-            background-position: 95% center;
-            background-color: transparent;
-            background-image: url("/icons/footer/arrow.svg");
-            background-repeat: no-repeat;
-            border: 0;
-            box-sizing: border-box;
-            color: white;
-            font-family: "Palanquin", sans-serif;
-            font-size: 14px;
-            font-weight: 800;
-            min-width: 90px;
-            padding-right: 25px;
-            text-transform: uppercase;
-            transition: background-position 0.1s;
-            width: 90px;
-            &:hover {
-              background-position: 100% center;
-              cursor: pointer;
-            }
+          @media (max-width: 768px) {
+            position: absolute;
+            bottom: 190px;
           }
         }
-      }
-      & .footer-logos {
-        display: flex;
-        justify-content: space-between;
-        margin-top: 25px;
-        & a {
-          display: inline-block;
-          & img {
-            height: 20px;
+        & .email-btn {
+          background-position: 90% center;
+          background-color: transparent;
+          background-image: url("/icons/footer/arrow.svg");
+          background-repeat: no-repeat;
+          border: 0;
+          box-sizing: border-box;
+          color: white;
+          font-family: "Palanquin", sans-serif;
+          font-size: 14px;
+          font-weight: 800;
+          min-width: 90px;
+          padding-right: 25px;
+          text-transform: uppercase;
+          transition: background-position 0.1s;
+          width: 90px;
+          &:hover {
+            background-position: 100% center;
+            cursor: pointer;
+          }
+          @media (max-width: 768px) {
+            display: flex;
+            margin: 0 auto;
+            justify-content: center;
           }
         }
       }
     }
   }
+  & .footer-logos {
+    background-color: #404A594D;
+    margin: 25px 0px 0px;
+    @media (max-width: 768px) {
+      padding-top: 5px;
+      padding-bottom: 5px;
+    }
+    & a {
+      display: inline-block;
+      margin: 20px;
+      @media (max-width: 768px) {
+        display: block;
+      }
+      & img {
+        height: 20px;
+      }
```

Comment:
The `#datawheel` logo needs to be taller 😏 

Because the icon in our logo is taller than the text, if both logo images have the same exact height our logo text ends up appearing smaller, and the whole thing feels unbalanced.